plugins/com.aptana.samples.ui/src/com/aptana/samples/ui/portal/actionController/SamplesActionController.java

```diff
@@ -256,7 +257,7 @@ public Object addSample(Object attributes)
 		if (natures == null || natures.length < 1)
 		{
 			natures = new String[1];
-			natures[0] = WEB_NATURE;
+			natures[0] = MOBILE_NATURE;
```

Comment:
In order to provide backward compatibility to Aptana Studio, let's add Web Nature as well.